logtag/logtag.go

```diff
@@ -0,0 +1,8 @@
+package logtag
```

Comment:
Multiple services will be logging. Please move any logging feature to a new repo. https://github.com/faraonc/hwsc-logging

Reply:
We can add this as a `dep` package.

Reply:
I can already predict logging will be big. We can hook it up with something like https://prometheus.io/ or https://grafana.com/ later.